tidb-cloud/tidb-cloud-upload-local-files.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Import Local Files to TiDB Cloud
+summary: Learn how to import local files to TiDB Cloud.
+---
+
+# Import Local Files to TiDB Cloud
+
+You can import local files to TiDB Cloud directly. You only need to select a local file, select the target cluster and table, make changes if needed, and then directly import the data into TiDB Cloud.
+
+This feature now supports the following:
+
+- Importing one CSV file within 50 MiB for one task
+- Only Serverless Tier clusters
+
+## Prerequisites
+
+- Prepare the local file to be imported. The file must be in CSV format within 50 MiB.
+- Use [SQL Editor](/develop/dev-guide-tidb-crud-sql.md#explore-sql-with-tidb) to create a table in the target database in advance. The table schema must be consistent with the local file. Alternatively, you can enter the table name in the import task and let TiDB Cloud help you automatically create the target table.
+
+## Import local files
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters), and choose your target project on the top of the left navigation bar.
+
+2. Locate your target cluster, click ... in the upper-right corner of the cluster area, and select **Import Data**.
+
+3. On the **Import** page, click the **Import Data** button in the upper-right corner, and then select **From local**.
+
+4. Click the **Upload File** button to select and upload the local file. One task only supports uploading one CSV file within 50 MiB.
```

Comment:
```suggestion
4. Click the **Upload File** button to select and upload the local file. Note that the file name must have a “。csv” suffix and must be less than 50 MiB.
```

Reply:
There are Chinese characters in it:  "“。"